pom.xml

```diff
@@ -44,10 +44,12 @@
     <camel.version>4.1.0</camel.version>
     <spring-boot.version>3.1.4</spring-boot.version>
     <xmlRpcClientVersion>3.1.3</xmlRpcClientVersion>
+    <fhir-stu4.version>7.0.0</fhir-stu4.version>
```

Comment:
FHIR releases post 3 use `R<x>` format. STU is "Standard for Trial Use"—essentially the specification version of "beta".

```suggestion
    <fhir-r4.version>7.0.0</fhir-r4.version>
```

Reply:
Thanks for pointing this out. 
Lets fix this in eip-erpnext-openmrs repo as well. https://github.com/ozone-his/eip-erpnext-openmrs/blob/main/pom.xml#L61

cc: @corneliouzbett